Relativity.Imaging.SDK.md

```diff
@@ -4,7 +4,7 @@
 
 This package contains interfaces for the public APIs of the Relativity Imaging application.
 
-## v12.1.0

```

Comment:
Is this a fix or a new release?  If it's a fix, this is fine, but if it's a new release we like to create a list such as:
https://github.com/relativitydev/relativity-api-documentation/blob/master/Relativity.Productions.SDK.md